src/SlateAsInputEditor/index.js

```diff
@@ -63,20 +62,45 @@ const ToolbarWrapper = styled.div`
   box-shadow: ${props => props.TOOLBAR_SHADOW || 'none'};
 `;
 
-const Heading = ({ type, children, attributes }) => createElement(
-  styled(type)`
-      font-family: 'serif';
-    `,
-  {
-    'data-key': attributes['data-key']
-  },
-  children,
-);
+const Heading = styled.div`
+  font-family: serif;
+`
 
 Heading.propTypes = {
   type: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6']),
 };
 
+const Code = styled.code(({ customCssObject }) => ({ ...customCssObject }))
```

Comment:
We've decided to remove styling props for code, pre, and blockquote, so please remove all code related to this to match the current state of the master branch.

Reply:
Sorry, I was having a problem signing the commits previously so I had to reopen a new PR and I mistakenly pulled the stale code also.
Just fixed it now.